suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -148,3 +150,15 @@ export const selectAccountTransactions = (
     state: TransactionsRootState,
     accountKey: string | null,
 ) => state.wallet.transactions.transactions[accountKey ?? ''] ?? [];
+
+export const selectAllTransactions = createSelector(
```

Comment:
Maybe rename it to `selectTransactionsData` as it returns this? I was confused at first about the difference between `selectTransactions` and `selectAllTransactions` when I read the code.

Reply:
I would rather go with renaming `selectTransactions` to `selectTransactionByAccount(Hash)` as you suggested

Reply:
Currently we have 
- `selectTransactions` - returns initial transactions setup from store
- `selectAccountTransactions` - selects transactions by accountKey
- `selectAllTransactions` - flat list of transactions

That seems fine to me. Maybe just add the param accountKey to the second one then?